environment/src/test/java/com/sequenceiq/environment/environment/validation/validators/TagValidatorTest.java

```diff
@@ -35,7 +35,7 @@
 
     public static final String BAD_KEY = "azureprefix";
```

Comment:
I would extend this or I would add more different bad values to cover more cases mentioned in links in the ticket.

Reply:
I agree, also not only Azure should be tested. But I'm not sure if it should be within the scope of this ticket.

Reply:
I think if we don't extend it here in this PR, no one will later.

Reply:
Updated tests as agreed.

Reply:
Thank you!